include/macros.h

```diff
@@ -26,12 +26,19 @@
 #define SQ(x) ((x)*(x))
 #define DECR(x) ((x) == 0 ? 0 : ((x) -= 1))
 
-extern GraphicsContext* oGfxCtx;
+extern GraphicsContext* __gfxCtx;
```

Comment:
Is this `extern` necessary?

Reply:
Yes, or else it will emit a pointer to .data